### ISSUE TITLE
Accessing Thread Local Storage

### ISSUE THREAD

**Issue by nightlark** [Robot/node-robot#7]
I'm looking for a way to access the TLS for a process from node, and didn't see anything in the API about it - maybe I just wasn't looking in the right place or there is some different terminology used, but the ability to access TLS seems like it would fit well with some of the other memory access functions robot-js provides.

The following SO question has a code snippet showing how it works on Windows. Linux and OS X seem to have their own variants.
http://reverseengineering.stackexchange.com/questions/3139/how-can-i-find-the-thread-local-storage-tls-of-a-windows-process-thread

**Comment by dkrutsko** [comment]
Yes I know about TLS (at least on Windows) but what sort of functionality were you looking for specifically? It seems that a lot of the functionality offered by TLS is not meant to be accessed directly and is probably outside the scope of robot-js. For these types of complex requirements, you may be better off using C++ directly, or writing a node C++ plugin to handle whatever you're trying to do. May also be worth looking into [BlackBone ](https://github.com/DarthTon/Blackbone). Either way, let me know.
